dietpi/dietpi-software

```diff
@@ -12318,6 +12324,40 @@ _EOF_
 			Download_Install "$(curl -sSfL 'https://api.github.com/repos/rclone/rclone/releases/latest' | mawk -F\" "/\"browser_download_url\": .*\/rclone-v[^\"\/]*linux-$arch.deb\"/{print \$4}")"
 
 		fi
+		
+		software_id=206 # Homer
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+		
+		Banner_Installing
+		
+		# installing nginx and ensuring it starts at boot
+		sudo apt install nginx -y
+		sudo systemctl enable nginx
+		
+		# installing homer
+		wget "https://github.com/bastienwirtz/homer/releases/latest/download/homer.zip" 
+		sudo unzip homer.zip -d /mnt/dietpi_userdata/Homer/
+		sudo cp /usr/share/nginx/homer/assets/config.yml.dist /mnt/dietpi_userdata/Homer/assets/config.yml
+		
+		#create nginx config - echo line by line to conf
+		sudo echo "server {" >> /etc/nginx/conf.d/homer.conf
+		sudo echo "    listen 49157;" >> /etc/nginx/conf.d/homer.conf
+		sudo echo "    listen [::]:49157;" >> /etc/nginx/conf.d/homer.conf
+		sudo echo "    root /mnt/dietpi_userdata/Homer/;" >> /etc/nginx/conf.d/homer.conf
+		sudo echo "   access_log /var/log/nginx/homer.access;" >> /etc/nginx/conf.d/homer.conf
+		sudo echo "   error_log /var/log/nginx/homer.error;" >> /etc/nginx/conf.d/homer.conf
+		sudo echo "}" >> /etc/nginx/conf.d/homer.conf
+		
+		
+		#fix permissions
+		sudo chown www-data:www-data /usr/share/nginx/homer/ -R
+		
+		#run
+		sudo systemctl reload nginx
+		
+		echo "installed homer, you may access it from YOUR_PRIVATE_IP_HERE:49157"
```

Comment:
For notification, use our `G_DIETPI-NOTIFY` function

Reply:
i'll add this